research-software-directory/RSD-as-a-service
# 399 sortable team members list

# Allow custom sorting of team members

Closes #399 

Changes proposed in this pull request:
*  Team members are by default sorted on name. The admin of the project can change the order by dragging the team members. The order is reflected on the project page
* Added drag icon to indicate that list items are "draggable" (the order can be manually changed) to team, testimonials (software) and custom pages

How to test:
* `docker-compose down --volumes` to remove old structure
* `docker-compose build` to build containers
* `docker-compose up --scale data-generation=1 --scale scrapers=0` to start solution and create dummy records
*  navigate to localhost 
*  login as rsd admin or create new project so you can edit it
* navigate to team section, add few members. 
* Change member order by dragging person up/down
* Verify the order is applied on the project page

## Example testimonials

![image](https://user-images.githubusercontent.com/9204081/193268505-574ce57d-2a08-4da7-af7e-15185fab4c4e.png)

## Example team
![image](https://user-images.githubusercontent.com/9204081/193268648-d5b90270-bef7-4f8d-a7a6-40bbef6cd7bf.png)

## Example custom pages
![image](https://user-images.githubusercontent.com/9204081/193268755-66b6fc9b-4529-43c6-8f51-d0087c612a09.png)

PR Checklist:
